ui/src/containers/VolumePageContent.js

```diff
@@ -131,39 +182,53 @@ const VolumePageContent = (props) => {
-            storageClassName={volume?.spec?.storageClassName}
-            creationTimestamp={volume?.metadata?.creationTimestamp}
-            volumeType={
-              volume?.spec?.rawBlockDevice
-                ? RAW_BLOCK_DEVICE
-                : SPARSE_LOOP_DEVICE
-            }
-            usedPodName={UsedPod ? UsedPod?.name : intl.translate('not_used')}
-            devicePath={
-              volume?.spec?.rawBlockDevice?.devicePath ??
-              intl.translate('not_applicable')
-            }
-            volumeUsagePercentage={currentVolume?.usage}
-            volumeUsageBytes={currentVolume?.usageRawData ?? 0}
-            storageCapacity={
-              volumeListData?.find((vol) => vol.name === currentVolumeName)
-                .storageCapacity
-            }
-            health={
-              volumeListData?.find((vol) => vol.name === currentVolumeName)
-                .health
+          <Tabs activeColor={theme.brand.primary} items={tabsItems}>
+            {
+              isOverviewPage &&
+              <VolumeDetailCard
+                name={currentVolumeName}
+                nodeName={volume?.spec?.nodeName}
+                storage={pV?.spec?.capacity?.storage ?? intl.translate('unknown')}
+                status={volumeStatus ?? intl.translate('unknown')}
+                storageClassName={volume?.spec?.storageClassName}
+                creationTimestamp={volume?.metadata?.creationTimestamp}
+                volumeType={
+                  volume?.spec?.rawBlockDevice
+                    ? RAW_BLOCK_DEVICE
+                    : SPARSE_LOOP_DEVICE
+                }
+                usedPodName={UsedPod ? UsedPod?.name : intl.translate('not_used')}
+                devicePath={
+                  volume?.spec?.rawBlockDevice?.devicePath ??
+                  intl.translate('not_applicable')
+                }
+                volumeUsagePercentage={currentVolume?.usage}
+                volumeUsageBytes={currentVolume?.usageRawData ?? 0}
+                storageCapacity={
+                  volumeListData?.find((vol) => vol.name === currentVolumeName)
+                    .storageCapacity
+                }
+                health={
+                  volumeListData?.find((vol) => vol.name === currentVolumeName)
+                    .health
+                }
+                condition={currentVolume.status}
+                // the delete button inside the volume detail card should know that which volume is the first one
+                volumeListData={volumeListData}
+                pVList={pVList}
+                alertlist={alertlist}
+              />
             }
-            condition={currentVolume.status}
-            // the delete button inside the volume detail card should know that which volume is the first one
-            volumeListData={volumeListData}
-            pVList={pVList}
-          ></VolumeDetailCard>
-          <ActiveAlertsCard
+            {
```

Comment:
We could have Switch inside the `<Tab>`, would you think it's easier to read?

```javascript
<Switch>
  <Route
    path={`/volumes/${volumeName}/overview`}
    component={VolumeOverviewTab}
   />
  ...
</Switch>
```

Reply:
Honestly, I prefer the 
`isOverviewPage &&`
approach.

We have to define the isXXXXPage variables anyway to tell the Tabs component which one is selected so I think we should as well use these variables to decide what to render since these are already defined.
It avoid to create more wrappers with the Route Switch and also IMO is easier to read.

But I totally understand if you prefer the Switch and want me to follow the existing coding-style. Let me know if I should change ? :)

Reply:
Yes. So let's follow the current coding-style ;)